lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py

```diff
@@ -51,11 +51,7 @@ def vetext_incoming_forwarder_lambda_handler(event: dict, context: any):
             if response is None:
                 push_to_retry_sqs(event_body)
 
-            responses.append(response)
-
-        logger.debug(responses)
-
-        return create_twilio_response(200)
+        return create_twilio_response()
```

Comment:
This might return None.  Is that correct?  I assume you want to return a status code.